2018-09-10 09:51:39,597-[main]-{INFO }-orca-(Globals.java:209)-Orca starting...
2018-09-10 09:51:39,599-[main]-{INFO }-orca-(Globals.java:210)-Home directory: /Users/komalthareja/renci/code/bug207/orca5/core/shirako/./../../core/shirako/net/exogeni/orca/
2018-09-10 09:51:39,599-[main]-{DEBUG}-orca-(Globals.java:191)-Registering Bouncy Castle Security Provider
2018-09-10 09:51:39,653-[main]-{INFO }-orca-(Globals.java:214)-Starting main container initialization...
2018-09-10 09:51:39,653-[main]-{INFO }-orca-(Globals.java:235)-Loading container configuration: path=/Users/komalthareja/renci/code/bug207/orca5/core/shirako/./../../core/shirako/net/exogeni/orca/config/orca.properties
2018-09-10 09:51:39,654-[main]-{INFO }-orca-(Globals.java:252)-Container configuration file loaded successfully.
2018-09-10 09:51:39,654-[main]-{INFO }-orca-(Globals.java:222)-Main container initialization complete.
2018-09-10 09:51:39,662-[main]-{INFO }-orca-(OrcaConfiguration.java:320)-axis2ClientRepo=/Users/komalthareja/renci/code/bug207/orca5/core/shirako/./../../core/shirako/net/exogeni/orca//axis2repository
2018-09-10 09:51:39,662-[main]-{DEBUG}-orca-(OrcaConfiguration.java:361)-Ticket factory class: net.exogeni.orca.shirako.common.delegation.SharpResourceTicketFactory
2018-09-10 09:51:39,670-[main]-{INFO }-orca-(Globals.java:117)-Container implementation class name: net.exogeni.orca.shirako.container.OrcaContainer
2018-09-10 09:51:39,676-[main]-{INFO }-orca-(Globals.java:120)-Successfully instantiated the container implementation.
2018-09-10 09:51:39,677-[main]-{INFO }-orca-(Globals.java:121)-Initializing container
2018-09-10 09:51:39,679-[main]-{DEBUG}-orca-(OrcaContainer.java:296)-Checking if this container is recovering. Looking for: /Users/komalthareja/renci/code/bug207/orca5/core/shirako/./../../core/shirako/net/exogeni/orca/state_recovery.lock
2018-09-10 09:51:39,679-[main]-{DEBUG}-orca-(OrcaContainer.java:301)-Superblock file does not exist. This is a fresh container
2018-09-10 09:51:39,680-[main]-{INFO }-orca-(OrcaContainer.java:307)-Creating container database
2018-09-10 09:51:39,711-[main]-{FATAL}-orca-(Globals.java:143)-Critical error: Orca failed to initialize
net.exogeni.orca.shirako.container.ContainerInitializationException: net.exogeni.orca.util.OrcaException: Cannot initialize due to Could not find default pool configuration
	at net.exogeni.orca.shirako.container.OrcaContainer.createDatabase(OrcaContainer.java:325)
	at net.exogeni.orca.shirako.container.OrcaContainer.initialize(OrcaContainer.java:227)
	at net.exogeni.orca.shirako.container.Globals.start(Globals.java:122)
	at net.exogeni.orca.shirako.container.OrcaTestCaseBase.fixClassPath(OrcaTestCaseBase.java:30)
	at net.exogeni.orca.shirako.container.OrcaTestCaseBase.<clinit>(OrcaTestCaseBase.java:23)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at junit.framework.TestSuite.createTest(TestSuite.java:61)
	at junit.framework.TestSuite.addTestMethod(TestSuite.java:294)
	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:150)
	at junit.framework.TestSuite.<init>(TestSuite.java:129)
	at org.junit.internal.runners.JUnit38ClassRunner.<init>(JUnit38ClassRunner.java:71)
	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:14)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:57)
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:29)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:57)
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:24)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:362)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:272)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:236)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
Caused by: net.exogeni.orca.util.OrcaException: Cannot initialize due to Could not find default pool configuration
	at net.exogeni.orca.util.db.MySqlBase.initialize(MySqlBase.java:337)
	at net.exogeni.orca.shirako.container.db.OrcaContainerDatabase.initialize(OrcaContainerDatabase.java:84)
	at net.exogeni.orca.shirako.container.OrcaContainer.createDatabase(OrcaContainer.java:322)
	... 25 more
Caused by: java.lang.RuntimeException: Could not find default pool configuration
	at net.exogeni.orca.util.db.MySqlPool.getDefaultConfiguration(MySqlPool.java:44)
	at net.exogeni.orca.util.db.MySqlPool.createConnectionPool(MySqlPool.java:67)
	at net.exogeni.orca.util.db.MySqlPool.registerPool(MySqlPool.java:100)
	at net.exogeni.orca.util.db.MySqlBase.initialize(MySqlBase.java:326)
	... 27 more
2018-09-10 09:54:11,775-[main]-{INFO }-orca-(Globals.java:209)-Orca starting...
2018-09-10 09:54:11,777-[main]-{INFO }-orca-(Globals.java:210)-Home directory: /Users/komalthareja/renci/code/bug207/orca5/core/shirako/./../../core/shirako/net/exogeni/orca/
2018-09-10 09:54:11,778-[main]-{DEBUG}-orca-(Globals.java:191)-Registering Bouncy Castle Security Provider
2018-09-10 09:54:11,833-[main]-{INFO }-orca-(Globals.java:214)-Starting main container initialization...
2018-09-10 09:54:11,834-[main]-{INFO }-orca-(Globals.java:235)-Loading container configuration: path=/Users/komalthareja/renci/code/bug207/orca5/core/shirako/./../../core/shirako/net/exogeni/orca/config/orca.properties
2018-09-10 09:54:11,835-[main]-{INFO }-orca-(Globals.java:252)-Container configuration file loaded successfully.
2018-09-10 09:54:11,835-[main]-{INFO }-orca-(Globals.java:222)-Main container initialization complete.
2018-09-10 09:54:11,845-[main]-{INFO }-orca-(OrcaConfiguration.java:320)-axis2ClientRepo=/Users/komalthareja/renci/code/bug207/orca5/core/shirako/./../../core/shirako/net/exogeni/orca//axis2repository
2018-09-10 09:54:11,845-[main]-{DEBUG}-orca-(OrcaConfiguration.java:361)-Ticket factory class: net.exogeni.orca.shirako.common.delegation.SharpResourceTicketFactory
2018-09-10 09:54:11,851-[main]-{INFO }-orca-(Globals.java:117)-Container implementation class name: net.exogeni.orca.shirako.container.OrcaContainer
2018-09-10 09:54:11,857-[main]-{INFO }-orca-(Globals.java:120)-Successfully instantiated the container implementation.
2018-09-10 09:54:11,858-[main]-{INFO }-orca-(Globals.java:121)-Initializing container
2018-09-10 09:54:11,860-[main]-{DEBUG}-orca-(OrcaContainer.java:296)-Checking if this container is recovering. Looking for: /Users/komalthareja/renci/code/bug207/orca5/core/shirako/./../../core/shirako/net/exogeni/orca/state_recovery.lock
2018-09-10 09:54:11,860-[main]-{DEBUG}-orca-(OrcaContainer.java:301)-Superblock file does not exist. This is a fresh container
2018-09-10 09:54:11,860-[main]-{INFO }-orca-(OrcaContainer.java:307)-Creating container database
2018-09-10 09:54:11,965-[main]-{FATAL}-orca-(Globals.java:143)-Critical error: Orca failed to initialize
net.exogeni.orca.shirako.container.ContainerInitializationException: net.exogeni.orca.util.OrcaException: Cannot initialize due to Cannot find configuration mapping file: orca/manage/container/db/map.container.mysql.xml
	at net.exogeni.orca.shirako.container.OrcaContainer.createDatabase(OrcaContainer.java:325)
	at net.exogeni.orca.shirako.container.OrcaContainer.initialize(OrcaContainer.java:227)
	at net.exogeni.orca.shirako.container.Globals.start(Globals.java:122)
	at net.exogeni.orca.shirako.container.OrcaTestCaseBase.fixClassPath(OrcaTestCaseBase.java:30)
	at net.exogeni.orca.shirako.container.OrcaTestCaseBase.<clinit>(OrcaTestCaseBase.java:23)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at junit.framework.TestSuite.createTest(TestSuite.java:61)
	at junit.framework.TestSuite.addTestMethod(TestSuite.java:294)
	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:150)
	at junit.framework.TestSuite.<init>(TestSuite.java:129)
	at org.junit.internal.runners.JUnit38ClassRunner.<init>(JUnit38ClassRunner.java:71)
	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:14)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:57)
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:29)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:57)
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:24)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:362)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:272)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:236)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
Caused by: net.exogeni.orca.util.OrcaException: Cannot initialize due to Cannot find configuration mapping file: orca/manage/container/db/map.container.mysql.xml
	at net.exogeni.orca.util.db.MySqlBase.initialize(MySqlBase.java:337)
	at net.exogeni.orca.shirako.container.db.OrcaContainerDatabase.initialize(OrcaContainerDatabase.java:84)
	at net.exogeni.orca.shirako.container.OrcaContainer.createDatabase(OrcaContainer.java:322)
	... 25 more
Caused by: java.io.FileNotFoundException: Cannot find configuration mapping file: orca/manage/container/db/map.container.mysql.xml
	at net.exogeni.orca.util.db.MySqlPropertiesMapper.readMysqlMapping(MySqlPropertiesMapper.java:256)
	at net.exogeni.orca.util.db.MySqlPropertiesMapper.createMapping(MySqlPropertiesMapper.java:59)
	at net.exogeni.orca.util.db.PropertiesMapper.initialize(PropertiesMapper.java:214)
	at net.exogeni.orca.util.db.MySqlBase.initialize(MySqlBase.java:329)
	... 27 more
2018-09-10 10:00:33,180-[main]-{INFO }-orca-(Globals.java:209)-Orca starting...
2018-09-10 10:00:33,182-[main]-{INFO }-orca-(Globals.java:210)-Home directory: /Users/komalthareja/renci/code/bug207/orca5/core/shirako/./../../core/shirako/net/exogeni/orca/
2018-09-10 10:00:33,182-[main]-{DEBUG}-orca-(Globals.java:191)-Registering Bouncy Castle Security Provider
2018-09-10 10:00:33,247-[main]-{INFO }-orca-(Globals.java:214)-Starting main container initialization...
2018-09-10 10:00:33,247-[main]-{INFO }-orca-(Globals.java:235)-Loading container configuration: path=/Users/komalthareja/renci/code/bug207/orca5/core/shirako/./../../core/shirako/net/exogeni/orca/config/orca.properties
2018-09-10 10:00:33,248-[main]-{INFO }-orca-(Globals.java:252)-Container configuration file loaded successfully.
2018-09-10 10:00:33,248-[main]-{INFO }-orca-(Globals.java:222)-Main container initialization complete.
2018-09-10 10:00:33,258-[main]-{INFO }-orca-(OrcaConfiguration.java:320)-axis2ClientRepo=/Users/komalthareja/renci/code/bug207/orca5/core/shirako/./../../core/shirako/net/exogeni/orca//axis2repository
2018-09-10 10:00:33,259-[main]-{DEBUG}-orca-(OrcaConfiguration.java:361)-Ticket factory class: net.exogeni.orca.shirako.common.delegation.SharpResourceTicketFactory
2018-09-10 10:00:33,266-[main]-{INFO }-orca-(Globals.java:117)-Container implementation class name: net.exogeni.orca.shirako.container.OrcaContainer
2018-09-10 10:00:33,275-[main]-{INFO }-orca-(Globals.java:120)-Successfully instantiated the container implementation.
2018-09-10 10:00:33,275-[main]-{INFO }-orca-(Globals.java:121)-Initializing container
2018-09-10 10:00:33,277-[main]-{DEBUG}-orca-(OrcaContainer.java:296)-Checking if this container is recovering. Looking for: /Users/komalthareja/renci/code/bug207/orca5/core/shirako/./../../core/shirako/net/exogeni/orca/state_recovery.lock
2018-09-10 10:00:33,277-[main]-{DEBUG}-orca-(OrcaContainer.java:301)-Superblock file does not exist. This is a fresh container
2018-09-10 10:00:33,278-[main]-{INFO }-orca-(OrcaContainer.java:307)-Creating container database
2018-09-10 10:00:33,724-[main]-{FATAL}-orca-(Globals.java:143)-Critical error: Orca failed to initialize
net.exogeni.orca.shirako.container.ContainerInitializationException: net.exogeni.orca.util.OrcaException: Cannot initialize due to Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at net.exogeni.orca.shirako.container.OrcaContainer.createDatabase(OrcaContainer.java:325)
	at net.exogeni.orca.shirako.container.OrcaContainer.initialize(OrcaContainer.java:227)
	at net.exogeni.orca.shirako.container.Globals.start(Globals.java:122)
	at net.exogeni.orca.shirako.container.OrcaTestCaseBase.fixClassPath(OrcaTestCaseBase.java:30)
	at net.exogeni.orca.shirako.container.OrcaTestCaseBase.<clinit>(OrcaTestCaseBase.java:23)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at junit.framework.TestSuite.createTest(TestSuite.java:61)
	at junit.framework.TestSuite.addTestMethod(TestSuite.java:294)
	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:150)
	at junit.framework.TestSuite.<init>(TestSuite.java:129)
	at org.junit.internal.runners.JUnit38ClassRunner.<init>(JUnit38ClassRunner.java:71)
	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:14)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:57)
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:29)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:57)
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:24)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:362)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:272)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:236)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
Caused by: net.exogeni.orca.util.OrcaException: Cannot initialize due to Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at net.exogeni.orca.util.db.MySqlBase.initialize(MySqlBase.java:337)
	at net.exogeni.orca.shirako.container.db.OrcaContainerDatabase.initialize(OrcaContainerDatabase.java:84)
	at net.exogeni.orca.shirako.container.OrcaContainer.createDatabase(OrcaContainer.java:322)
	... 25 more
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:989)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:341)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2196)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2229)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2024)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:779)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:389)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:330)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:75)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1148)
	at org.apache.commons.dbcp.PoolingDriver.connect(PoolingDriver.java:180)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at net.exogeni.orca.util.db.MySqlBase.getConnection(MySqlBase.java:256)
	at net.exogeni.orca.util.db.MySqlBase.checkDb(MySqlBase.java:217)
	at net.exogeni.orca.util.db.MySqlBase.initialize(MySqlBase.java:332)
	... 27 more
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:211)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:300)
	... 50 more
2018-09-10 10:05:12,872-[main]-{INFO }-orca-(Globals.java:209)-Orca starting...
2018-09-10 10:05:12,875-[main]-{INFO }-orca-(Globals.java:210)-Home directory: /Users/komalthareja/renci/code/bug207/orca5/core/shirako/./../../core/shirako/net/exogeni/orca/
2018-09-10 10:05:12,875-[main]-{DEBUG}-orca-(Globals.java:191)-Registering Bouncy Castle Security Provider
2018-09-10 10:05:13,008-[main]-{INFO }-orca-(Globals.java:214)-Starting main container initialization...
2018-09-10 10:05:13,009-[main]-{INFO }-orca-(Globals.java:235)-Loading container configuration: path=/Users/komalthareja/renci/code/bug207/orca5/core/shirako/./../../core/shirako/net/exogeni/orca/config/orca.properties
2018-09-10 10:05:13,009-[main]-{INFO }-orca-(Globals.java:252)-Container configuration file loaded successfully.
2018-09-10 10:05:13,010-[main]-{INFO }-orca-(Globals.java:222)-Main container initialization complete.
2018-09-10 10:05:13,025-[main]-{INFO }-orca-(OrcaConfiguration.java:320)-axis2ClientRepo=/Users/komalthareja/renci/code/bug207/orca5/core/shirako/./../../core/shirako/net/exogeni/orca//axis2repository
2018-09-10 10:05:13,025-[main]-{DEBUG}-orca-(OrcaConfiguration.java:361)-Ticket factory class: net.exogeni.orca.shirako.common.delegation.SharpResourceTicketFactory
2018-09-10 10:05:13,037-[main]-{INFO }-orca-(Globals.java:117)-Container implementation class name: net.exogeni.orca.shirako.container.OrcaContainer
2018-09-10 10:05:13,073-[main]-{INFO }-orca-(Globals.java:120)-Successfully instantiated the container implementation.
2018-09-10 10:05:13,073-[main]-{INFO }-orca-(Globals.java:121)-Initializing container
2018-09-10 10:05:13,075-[main]-{DEBUG}-orca-(OrcaContainer.java:296)-Checking if this container is recovering. Looking for: /Users/komalthareja/renci/code/bug207/orca5/core/shirako/./../../core/shirako/net/exogeni/orca/state_recovery.lock
2018-09-10 10:05:13,075-[main]-{DEBUG}-orca-(OrcaContainer.java:301)-Superblock file does not exist. This is a fresh container
2018-09-10 10:05:13,075-[main]-{INFO }-orca-(OrcaContainer.java:307)-Creating container database
2018-09-10 10:05:13,603-[main]-{FATAL}-orca-(Globals.java:143)-Critical error: Orca failed to initialize
net.exogeni.orca.shirako.container.ContainerInitializationException: net.exogeni.orca.util.OrcaException: Cannot initialize due to Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at net.exogeni.orca.shirako.container.OrcaContainer.createDatabase(OrcaContainer.java:325)
	at net.exogeni.orca.shirako.container.OrcaContainer.initialize(OrcaContainer.java:227)
	at net.exogeni.orca.shirako.container.Globals.start(Globals.java:122)
	at net.exogeni.orca.shirako.container.OrcaTestCaseBase.fixClassPath(OrcaTestCaseBase.java:30)
	at net.exogeni.orca.shirako.container.OrcaTestCaseBase.<clinit>(OrcaTestCaseBase.java:23)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at junit.framework.TestSuite.createTest(TestSuite.java:61)
	at junit.framework.TestSuite.addTestMethod(TestSuite.java:294)
	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:150)
	at junit.framework.TestSuite.<init>(TestSuite.java:129)
	at org.junit.internal.runners.JUnit38ClassRunner.<init>(JUnit38ClassRunner.java:71)
	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:14)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:57)
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:29)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:57)
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:24)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:362)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:272)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:236)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
Caused by: net.exogeni.orca.util.OrcaException: Cannot initialize due to Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at net.exogeni.orca.util.db.MySqlBase.initialize(MySqlBase.java:337)
	at net.exogeni.orca.shirako.container.db.OrcaContainerDatabase.initialize(OrcaContainerDatabase.java:84)
	at net.exogeni.orca.shirako.container.OrcaContainer.createDatabase(OrcaContainer.java:322)
	... 25 more
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:989)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:341)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2196)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2229)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2024)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:779)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:389)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:330)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:75)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1148)
	at org.apache.commons.dbcp.PoolingDriver.connect(PoolingDriver.java:180)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at net.exogeni.orca.util.db.MySqlBase.getConnection(MySqlBase.java:256)
	at net.exogeni.orca.util.db.MySqlBase.checkDb(MySqlBase.java:217)
	at net.exogeni.orca.util.db.MySqlBase.initialize(MySqlBase.java:332)
	... 27 more
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:211)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:300)
	... 50 more
2018-09-10 10:10:54,469-[main]-{INFO }-orca-(Globals.java:209)-Orca starting...
2018-09-10 10:10:54,471-[main]-{INFO }-orca-(Globals.java:210)-Home directory: /Users/komalthareja/renci/code/bug207/orca5/core/shirako/./../../core/shirako/net/exogeni/orca/
2018-09-10 10:10:54,472-[main]-{DEBUG}-orca-(Globals.java:191)-Registering Bouncy Castle Security Provider
2018-09-10 10:10:54,573-[main]-{INFO }-orca-(Globals.java:214)-Starting main container initialization...
2018-09-10 10:10:54,574-[main]-{INFO }-orca-(Globals.java:235)-Loading container configuration: path=/Users/komalthareja/renci/code/bug207/orca5/core/shirako/./../../core/shirako/net/exogeni/orca/config/orca.properties
2018-09-10 10:10:54,574-[main]-{INFO }-orca-(Globals.java:252)-Container configuration file loaded successfully.
2018-09-10 10:10:54,574-[main]-{INFO }-orca-(Globals.java:222)-Main container initialization complete.
2018-09-10 10:10:54,615-[main]-{INFO }-orca-(OrcaConfiguration.java:320)-axis2ClientRepo=/Users/komalthareja/renci/code/bug207/orca5/core/shirako/./../../core/shirako/net/exogeni/orca//axis2repository
2018-09-10 10:10:54,615-[main]-{DEBUG}-orca-(OrcaConfiguration.java:361)-Ticket factory class: net.exogeni.orca.shirako.common.delegation.SharpResourceTicketFactory
2018-09-10 10:10:54,626-[main]-{INFO }-orca-(Globals.java:117)-Container implementation class name: net.exogeni.orca.shirako.container.OrcaContainer
2018-09-10 10:10:54,632-[main]-{INFO }-orca-(Globals.java:120)-Successfully instantiated the container implementation.
2018-09-10 10:10:54,633-[main]-{INFO }-orca-(Globals.java:121)-Initializing container
2018-09-10 10:10:54,648-[main]-{DEBUG}-orca-(OrcaContainer.java:296)-Checking if this container is recovering. Looking for: /Users/komalthareja/renci/code/bug207/orca5/core/shirako/./../../core/shirako/net/exogeni/orca/state_recovery.lock
2018-09-10 10:10:54,649-[main]-{DEBUG}-orca-(OrcaContainer.java:301)-Superblock file does not exist. This is a fresh container
2018-09-10 10:10:54,649-[main]-{INFO }-orca-(OrcaContainer.java:307)-Creating container database
2018-09-10 10:10:55,137-[main]-{FATAL}-orca-(Globals.java:143)-Critical error: Orca failed to initialize
net.exogeni.orca.shirako.container.ContainerInitializationException: net.exogeni.orca.util.OrcaException: Cannot initialize due to Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at net.exogeni.orca.shirako.container.OrcaContainer.createDatabase(OrcaContainer.java:325)
	at net.exogeni.orca.shirako.container.OrcaContainer.initialize(OrcaContainer.java:227)
	at net.exogeni.orca.shirako.container.Globals.start(Globals.java:122)
	at net.exogeni.orca.shirako.container.OrcaTestCaseBase.fixClassPath(OrcaTestCaseBase.java:30)
	at net.exogeni.orca.shirako.container.OrcaTestCaseBase.<clinit>(OrcaTestCaseBase.java:23)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at junit.framework.TestSuite.createTest(TestSuite.java:61)
	at junit.framework.TestSuite.addTestMethod(TestSuite.java:294)
	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:150)
	at junit.framework.TestSuite.<init>(TestSuite.java:129)
	at org.junit.internal.runners.JUnit38ClassRunner.<init>(JUnit38ClassRunner.java:71)
	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:14)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:57)
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:29)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:57)
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:24)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:362)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:272)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:236)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
Caused by: net.exogeni.orca.util.OrcaException: Cannot initialize due to Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at net.exogeni.orca.util.db.MySqlBase.initialize(MySqlBase.java:337)
	at net.exogeni.orca.shirako.container.db.OrcaContainerDatabase.initialize(OrcaContainerDatabase.java:84)
	at net.exogeni.orca.shirako.container.OrcaContainer.createDatabase(OrcaContainer.java:322)
	... 25 more
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:989)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:341)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2196)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2229)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2024)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:779)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:389)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:330)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:75)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1148)
	at org.apache.commons.dbcp.PoolingDriver.connect(PoolingDriver.java:180)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at net.exogeni.orca.util.db.MySqlBase.getConnection(MySqlBase.java:256)
	at net.exogeni.orca.util.db.MySqlBase.checkDb(MySqlBase.java:217)
	at net.exogeni.orca.util.db.MySqlBase.initialize(MySqlBase.java:332)
	... 27 more
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:211)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:300)
	... 50 more
2018-09-10 10:31:15,342-[main]-{INFO }-orca-(Globals.java:209)-Orca starting...
2018-09-10 10:31:15,344-[main]-{INFO }-orca-(Globals.java:210)-Home directory: /Users/komalthareja/renci/code/bug207/orca5/core/shirako/./../../core/shirako/net/exogeni/orca/
2018-09-10 10:31:15,345-[main]-{DEBUG}-orca-(Globals.java:191)-Registering Bouncy Castle Security Provider
2018-09-10 10:31:15,410-[main]-{INFO }-orca-(Globals.java:214)-Starting main container initialization...
2018-09-10 10:31:15,411-[main]-{INFO }-orca-(Globals.java:235)-Loading container configuration: path=/Users/komalthareja/renci/code/bug207/orca5/core/shirako/./../../core/shirako/net/exogeni/orca/config/orca.properties
2018-09-10 10:31:15,411-[main]-{INFO }-orca-(Globals.java:252)-Container configuration file loaded successfully.
2018-09-10 10:31:15,411-[main]-{INFO }-orca-(Globals.java:222)-Main container initialization complete.
2018-09-10 10:31:15,420-[main]-{INFO }-orca-(OrcaConfiguration.java:320)-axis2ClientRepo=/Users/komalthareja/renci/code/bug207/orca5/core/shirako/./../../core/shirako/net/exogeni/orca//axis2repository
2018-09-10 10:31:15,421-[main]-{DEBUG}-orca-(OrcaConfiguration.java:361)-Ticket factory class: net.exogeni.orca.shirako.common.delegation.SharpResourceTicketFactory
2018-09-10 10:31:15,427-[main]-{INFO }-orca-(Globals.java:117)-Container implementation class name: net.exogeni.orca.shirako.container.OrcaContainer
2018-09-10 10:31:15,438-[main]-{INFO }-orca-(Globals.java:120)-Successfully instantiated the container implementation.
2018-09-10 10:31:15,438-[main]-{INFO }-orca-(Globals.java:121)-Initializing container
2018-09-10 10:31:15,444-[main]-{DEBUG}-orca-(OrcaContainer.java:296)-Checking if this container is recovering. Looking for: /Users/komalthareja/renci/code/bug207/orca5/core/shirako/./../../core/shirako/net/exogeni/orca/state_recovery.lock
2018-09-10 10:31:15,445-[main]-{DEBUG}-orca-(OrcaContainer.java:301)-Superblock file does not exist. This is a fresh container
2018-09-10 10:31:15,445-[main]-{INFO }-orca-(OrcaContainer.java:307)-Creating container database
2018-09-10 10:31:15,928-[main]-{FATAL}-orca-(Globals.java:143)-Critical error: Orca failed to initialize
net.exogeni.orca.shirako.container.ContainerInitializationException: net.exogeni.orca.util.OrcaException: Cannot initialize due to Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at net.exogeni.orca.shirako.container.OrcaContainer.createDatabase(OrcaContainer.java:325)
	at net.exogeni.orca.shirako.container.OrcaContainer.initialize(OrcaContainer.java:227)
	at net.exogeni.orca.shirako.container.Globals.start(Globals.java:122)
	at net.exogeni.orca.shirako.container.OrcaTestCaseBase.fixClassPath(OrcaTestCaseBase.java:30)
	at net.exogeni.orca.shirako.container.OrcaTestCaseBase.<clinit>(OrcaTestCaseBase.java:23)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at junit.framework.TestSuite.createTest(TestSuite.java:61)
	at junit.framework.TestSuite.addTestMethod(TestSuite.java:294)
	at junit.framework.TestSuite.addTestsFromTestCase(TestSuite.java:150)
	at junit.framework.TestSuite.<init>(TestSuite.java:129)
	at org.junit.internal.runners.JUnit38ClassRunner.<init>(JUnit38ClassRunner.java:71)
	at org.junit.internal.builders.JUnit3Builder.runnerForClass(JUnit3Builder.java:14)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:57)
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:29)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:57)
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:24)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:362)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:272)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:236)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
Caused by: net.exogeni.orca.util.OrcaException: Cannot initialize due to Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at net.exogeni.orca.util.db.MySqlBase.initialize(MySqlBase.java:337)
	at net.exogeni.orca.shirako.container.db.OrcaContainerDatabase.initialize(OrcaContainerDatabase.java:84)
	at net.exogeni.orca.shirako.container.OrcaContainer.createDatabase(OrcaContainer.java:322)
	... 25 more
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:989)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:341)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2196)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2229)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2024)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:779)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:389)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:330)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:75)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:1148)
	at org.apache.commons.dbcp.PoolingDriver.connect(PoolingDriver.java:180)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at net.exogeni.orca.util.db.MySqlBase.getConnection(MySqlBase.java:256)
	at net.exogeni.orca.util.db.MySqlBase.checkDb(MySqlBase.java:217)
	at net.exogeni.orca.util.db.MySqlBase.initialize(MySqlBase.java:332)
	... 27 more
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:211)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:300)
	... 50 more
